version.go

```diff
@@ -4,4 +4,4 @@
 package instana
 
 // Version is the version of Instana sensor
-const Version = "1.28.1"
+const Version = "1.29.0"
```

Comment:
So far we've been bumping the version by pushing the commit directly to `master` before the release. I'm fine with either though.

Reply:
I agree, that it should not make a big difference.